### PR TITLE
Task 428

### PR DESCRIPTION
**Pull request by RanimHelamiah**
finish task-24 in AC

### CODE REVIEW

File: lib/organisms/components/banners/banner1/style.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ const BannerMask = styled.div`
   width: 100%;
   height: 100%;
   min-height: 100vh;
-  padding-top: 210px;
+  padding-top: 110px;
```
Comment (AMhaish) [inline lib/organisms/components/banners/banner1/style.ts:58]: Just wanted to double-check with you about this change while the original was 110px

File: lib/organisms/components/banners/banner1/index.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ const BannerSection = ({
       id={model.name}
     >
       <BannerMask>
-        <Particles />
+        {/* <Particles /> */}
```
Comment (AMhaish) [inline lib/organisms/components/banners/banner1/index.tsx:73]: Don't we have particles in the original theme here?

File: lib/organisms/components/banners/banner1/style.ts (hunk 1 above)
Comment (RanimHelamiah) [reply lib/organisms/components/banners/banner1/style.ts:58]: It was 210 pixels, but its size was too large for the link I was comparing on. That's why I made it smaller so it became exactly the same.

File: lib/organisms/components/banners/banner1/index.tsx (hunk 2 above)
Comment (RanimHelamiah) [reply lib/organisms/components/banners/banner1/index.tsx:73]: But in a smaller size and shape than ours... I initially made them for a while and I hope you see the difference if we put them in, they will change the design.

**Pull request merged**